CMakeLists.txt

```diff
@@ -61,13 +61,16 @@ if (ENABLE_CHECK_HEAVY_BUILDS)
     set (RLIMIT_AS 10000000000)
     # set CPU time limit to 1000 seconds
     set (RLIMIT_CPU 1000)
+    # set stack size to 1024M
+    set (RLIMIT_STACK 1024000000)
```

Comment:
No wonder it does not work - clang simply assume that the stack is 8MB - https://github.com/llvm/llvm-project/blob/788b50a4384985f2c221cfd8d290cabc6f59e646/clang/include/clang/Basic/Stack.h#L26

As for why it happens only for arm64 it is likely due to it has more registers

And here is an example of a stacktrace - https://pastila.nl/?0000e83f/bcc54268487047ba960a50304e756557#7k0S6lGYXncNor/j2uTBqw==

Reply:
Introduced in [c978f0f7](https://github.com/llvm/llvm-project/commit/c978f0f7)

```
$ git tag --contains c978f0f7
llvmorg-21-init
llvmorg-20.1.3
llvmorg-20.1.2
llvmorg-20.1.1
llvmorg-20.1.0-rc3
llvmorg-20.1.0-rc2
llvmorg-20.1.0-rc1
llvmorg-20.1.0
```